lib/CleantalkSP/SpbctWP/DB/SQLSchema.php

```diff
@@ -176,6 +176,7 @@ class SQLSchema extends \CleantalkSP\Common\DB\SQLSchema
                 array('field' => 'bad_code', 'type' => 'tinyint', 'null' => 'yes',),
                 array('field' => 'csrf', 'type' => 'tinyint', 'null' => 'yes',),
                 array('field' => 'weak_spots', 'type' => 'varchar(2048)', 'null' => 'yes',),
+                array('field' => 'approved', 'type' => 'tinyint', 'null' => 'yes',),
```

Comment:
may be zero by default? it's easier than keeping NULL in addition to two values

Reply:
https://github.com/CleanTalk/security-malware-firewall/pull/109/commits/5a0b84cf14c79df028a55d67aace2156a6acb6cc Done